nix/mobile/android/build.nix

```
@@ -28,7 +28,7 @@ let
   inherit (lib) toLower optionalString stringLength makeLibraryPath elem;
 
   # Pass secretsFile for POKT_TOKEN to jsbundle build
-  builtJsBundle = jsbundle { inherit secretsFile; };
+  builtJsBundle =  lib.optionals (buildType != "debug") jsbundle { inherit secretsFile; };
```

You could define this in the `let` block for reuse in both places:
```suggestion
  ifDebug = lib.optionals (buildType != "debug");
  
  # Pass secretsFile for POKT_TOKEN to jsbundle build
  builtJsBundle = ifDebug jsbundle { inherit secretsFile; };
```

---

oh nice idea